index.html

```diff
@@ -17,12 +18,77 @@
 <body>
 
     <div class="container">
+
+     
       <div class="row">
           <div class="col-sm-12">
-              <h1>Start Here!</h1>
-          </div>
-      </div>
-    </div><!-- container -->
+             
+                  <nav class="nav nav-pills nav-justified">
+                  <a class="nav-item nav-link active" href="#">Home</a>
+                  <a class="nav-item nav-link" href="#">About</a>
+                  <a class="nav-item nav-link" href="#">Projects</a>
+                  <a class="nav-item nav-link" href="#">Contact</a>
+                </nav>
+                </div>
+                <div class="row">
+                  <div class="col-md-12">
+                    <div class="jumbotron jumbotron-fluid">
+                        <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSDKMfyMkydpP2j1jxAta4tJp3olRdDcarDIfUMGqRk5G_Wn6xW" class="img-fluid width:100%" alt="Responsive image">
+                           <h1 class="display-4">Boniface Makau- Programmer</h1>
+                           
+
+                    </div>
+
+                  </div>
+
+                </div>
+                <div>
+                  <div class="lead">Skills and Experiences</div>
+                <ul class="list-group list-group-flush">
+                    <li class="list-group-item">Bootstrap</li>
+                    <li class="list-group-item">JavaScript</li>
+                    <li class="list-group-item">CSS Preprocessor</li>
+                    <li class="list-group-item">Java</li>
+                    <li class="list-group-item">MongoDB</li>
+                  </ul>
+                </div>
+                <div class="row">
+                <table class="table table-bordered table-dark">
+                <thead>
+                  <tr>
+                    <th scope="col">Work Experience</th>
+                  </tr>
+                  <tr>
+                    <th scope="col">#</th>
+                    <th scope="col">Name of the Company</th>
+                    <th scope="col">Years worked</th>
+                    <th scope="col">Duties</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <th scope="row">1</th>
+                    <td>Zenith Bank</td>
+                    <td>2</td>
+                    <td>Web Development</td>
+                  </tr>
+                  <tr>
+                    <th scope="row">2</th>
+                    <td>Chase Bank</td>
+                    <td>4</td>
+                    <td>Android Developer</td>
+                  </tr>
+                  <tr>
+                      <th scope="row">3</th>
+                      <td>Huntington Enterprises</td>
+                      <td>6</td>
+                      <td>Android Developer</td>
+                    </tr>
```

Comment:
That's some really cool experience!  What languages did you use?  Was this primarily Java?